src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/summaryjavadoc/InputSummaryJavadocInlineCorrect.java

```diff
@@ -108,6 +108,22 @@ public int validInlineJavadocReturn(int a) // ok
      */
     private void voidValidJavadoc() {} // ok
 
+    /**
+     * {@return a zero}
```

Comment:
please move this input code to serparate file, we recently come to agrement to not have long Input files (120 lines max).
it will also allow you to avoid update in UT methods.
please do this for other Inputs also.

Reply:
done

Reply:
I still see changes in this file, please create new Input file.
sorry that it is not validated by build, this file is ignore list, but it will be split soon to follow common limit.

Reply:
apologies, done for sure this time.